backend/src/test/java/com/allog/dallog/domain/category/application/CategoryServiceTest.java

```diff
@@ -68,512 +63,482 @@
-        CategoryResponse 우아한테크코스_외부_일정 = categoryService.save(관리자.getId(), 우아한테크코스_외부_일정_생성_요청);
+        네오.회원_가입을_한다(네오_이메일, 네오_이름, 네오_프로필_URL)
+                .카테고리를_등록한다(외부_카테고리_이름, GOOGLE);
 
         // when
-        categoryService.delete(관리자.getId(), 우아한테크코스_외부_일정.getId());
+        categoryService.delete(네오.계정().getId(), 네오.카테고리().getId());
 
         // then
-        assertThatThrownBy(() -> categoryService.findDetailCategoryById(우아한테크코스_외부_일정.getId()))
+        assertThatThrownBy(() -> categoryService.findDetailCategoryById(네오.카테고리().getId()))
                 .isInstanceOf(NoSuchCategoryException.class);
     }
+
+    private final class User {
+
+        private Member member;
+        private Category category;
+        private CategoryRole categoryRole;
+        private Subscription subscription;
+        private Schedule schedule;
+
+        private User 회원_가입을_한다(final String email, final String name, final String profile) {
+            Member member = new Member(email, name, profile, SocialType.GOOGLE);
+            this.member = memberRepository.save(member);
+            return this;
+        }
+
+        private User 카테고리를_등록한다(final String categoryName, final CategoryType categoryType) {
+            Category category = new Category(categoryName, this.member, categoryType);
+            CategoryRole categoryRole = new CategoryRole(category, this.member, ADMIN);
+            Subscription subscription = new Subscription(this.member, category, COLOR_1);
+            this.category = categoryRepository.save(category);
+            this.categoryRole = categoryRoleRepository.save(categoryRole);
+            this.subscription = subscriptionRepository.save(subscription);
+            return this;
+        }
+
+        private User 카테고리를_구독한다(final Category category) {
+            Subscription subscription = new Subscription(this.member, category, COLOR_1);
+            CategoryRole categoryRole = new CategoryRole(category, this.member, NONE);
+            this.subscription = subscriptionRepository.save(subscription);
+            this.categoryRole = categoryRoleRepository.save(categoryRole);
+            return this;
+        }
+
+        private User 내_카테고리_관리_권한을_부여한다(final Member otherMember) {
+            CategoryRole categoryRole = categoryRoleRepository.getByMemberIdAndCategoryId(otherMember.getId(),
+                    category.getId());
+            categoryRole.changeRole(ADMIN);
+            categoryRoleRepository.save(categoryRole);
+            return this;
+        }
+
+        private User 일정을_등록한다(final String title, final LocalDateTime start, final LocalDateTime end,
+                              final String memo) {
+            Schedule schedule = new Schedule(this.category, title, start, end, memo);
+            this.schedule = scheduleRepository.save(schedule);
+            return this;
+        }
+
+        private Member 계정() {
+            return member;
+        }
```

Comment:
```suggestion
        private Member 회원() {
            return member;
        }
```

마찬가지로 도메인 용어를 사용하는게 좋아보여요!

Reply:
`회원()` 메서드 명으로 변경해보았어요!